lib/defaults/defaults.go

```diff
@@ -263,6 +263,18 @@ const (
 	// SatelliteRPCAgentPort is port used by satellite agent to expose its status
 	SatelliteRPCAgentPort = 7575
 
+	// SatelliteRPCAgentPort is port used by satellite agent to expose metrics
```

Comment:
I re-arranged the defaults to have constants and variables (and functions) separately - mostly moving things that did not belong in `var` block to `const`.